spinalcordtoolbox/deepseg/inference.py

```diff
@@ -233,13 +233,19 @@ def segment_nnunet(path_img, tmpdir, predictor, device: torch.device):
     orig_orientation = get_orientation(Image(path_img_tmp))
 
     # Get the orientation used by the model
-    # TODO: Make reorientation a model configuration parameter
-    if "SCI" in predictor.plans_manager.dataset_name:
-        model_orientation = "RPI"
-    elif "RegionBasedLesionSeg" in predictor.plans_manager.dataset_name:
-        model_orientation = "AIL"
+    # Check if predictor.dataset_json['orientation'] key exists, if so, read the orientation from there
```

Comment:
to exactly match [#4683](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/as/4683_add_canal_segmentation_nnunet/spinalcordtoolbox/deepseg/inference.py#L235-L246) and avoid conflicts

```suggestion
    # Check if predictor.dataset_json['image_orientation'] key exists, if so, read the orientation from there
```

Reply:
Good catch !